release notes/v0.38.0.md

```diff
@@ -0,0 +1,275 @@
+k6 v0.38.0 is here! 🎉
+
+## New Features!
+
+### Accessing the consolidated and derived options from the default function ([2493](https://github.com/grafana/k6/pull/2493))
```

Comment:
That's a good point, could you suggest something @codebien, please? 👀

Reply:
>Why would someone want to do this?

The motivation is like the title says _Accessing the consolidated and derived options_. In the currently released versions, all the available ways were hacks and workarounds and it hasn't been any guarantee in terms of API and consistency.

I hope the following example explains better the goal:

```javascript
import exec from 'k6/execution'

export const options = {
	vus: 1,
	iterations: 1,
  tags: { mytag: 'myvalue' },
};

export default function() {
	console.log(options.tags.mytag);
	console.log(exec.test.options.tags.mytag); // consolidated case 

	console.log(options.scenarios.default);
	console.log(exec.test.options.scenarios.default); // derived case
}

---
INFO[0000] undefined                                     source=console
INFO[0000] myvalue                                       source=console
INFO[0000] undefined                                     source=console
INFO[0000] {"executor":"shared-iterations","startTime":null,"gracefulStop":null,"env":null,"exec":null,"tags":null,"vus":1,"iterations":1,"maxDuration":null}  source=console
```

An additional real case where you can find more details could be this open issue https://github.com/grafana/k6/issues/2278 

@oleiade I think we could add also a link to the docs page where it will be available the documentation for this feature. https://github.com/grafana/k6-docs/pull/629
The link should be: https://k6.io/docs/javascript-api/k6-execution/#test

Reply:
I've added a link to the documentation section, indeed. If you folks wish to add an explanation as to which scenarios and workflows users should use this in, feel free to post a commitable suggestion. Otherwise, we'll resolve and leave it as it 👍🏻